ros/src/twist_controller/twist_controller_mod.py

```diff
@@ -124,6 +131,9 @@ def control(self, target_linear_velocity, target_angular_velocity, current_linea
         filtered_throttle = min(
             self.max_throttle, self.throttle_lpf.filt(throttle_command))
 
+        # Store final brake torque command for manual_braking_torque prev value
+        self.manual_braking_torque = filtered_brake
```

Comment:
Should we consider naming this instance variable `previous_manual_braking_torque`? I understand completely how it's used once I reviewed the code, but it wasn't clear immediately that it was a time-varying value rather than a configuration parameter.

Reply:
Good point, renamed it.